frontend/src/container/Sidebar.jsx

```diff
@@ -81,14 +82,15 @@ export class Sidebar extends React.Component {
     if (!this.shouldDisplaySidebar(props)) return
 
     if (
-      props.match.params &&
-      props.match.params.idws &&
-      props.workspaceList.find(ws => ws.isOpenInSidebar) === undefined
+      props.location.pathname.includes('workspaces/') &&
```

Comment:
having to hard code the string `workspaces` is technical dept.
Ideally it should come from the `PAGE` constant. But I understand that it would imply refactor the entire `PAGE` constant which might not be the time.
I think each parts of urls in the PAGE constant (eg. 'workspaces', 'content', 'admin' ...) should come from constants.

Creating a "technical dept" issue might be a good idea

Reply:
Updated. (I used the PAGE constant)